src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1967,6 +1967,12 @@ public static IEnumerable<DValue<RecordValue>> StandardTableNodeRecords(IRContex
                 else if (!forceSingleColumn && arg is BlankValue bv && tableType.FieldNames.Count() > 1)
```

Comment:
>tableType.FieldNames.Count() [](http://example.com/codeflow?start=71&length=28)

Can be cached outside the loop (e.g., `var fieldCount = tableType.FieldNames.Count();` after the declaration of tableType) since the value doesn't change